### PR TITLE
Create test instance model

### PR DESCRIPTION
**Pull request by zkan**
Changes proposed in this pull request:
* Create a test instance model that links to a test case
* Register the test instance model to Django admin (plus search by assignee)

### CODE REVIEW

File: api/prontolista/test_instances/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
+
+from model_bakery import baker
```
Comment (gatukgl) [inline api/prontolista/test_instances/tests/test_models.py]: ง้อว

File: api/prontolista/test_instances/models.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+from django_extensions.db.models import TimeStampedModel
+
+from testcases.models import TestCase
+
+
+class TestInstance(TimeStampedModel):
+    testcase = models.ForeignKey(TestCase, on_delete=models.CASCADE)
+    assignee = models.CharField(null=True, blank=True, max_length=300)
+    statuses = (("passed", "Passed"), ("blocked", "Blocked"), ("failed", "Failed"))
```
Comment (yothinix) [inline api/prontolista/test_instances/models.py]: This should be upper case STATUSES

at first glance, I thought it was another field until I realize it's a list of choice

File: api/prontolista/testcases/tests/test_models.py [hunk 3]
```diff
@@ -24,3 +24,10 @@ def test_model_should_have_foreign_key_to_project(self):
         assert actual.project.name == expected_project_name
         assert actual.created
         assert actual.modified
+
+    def test_model_should_get_name_as_string_representation(self):
+        project = mommy.make(Project)
```
Comment (yothinix) [inline api/prontolista/testcases/tests/test_models.py]: Shall we start migrate to use model_bakery here too?

File: api/prontolista/test_instances/models.py (hunk 2 above)
Comment (zkan) [reply api/prontolista/test_instances/models.py]: Okie

File: api/prontolista/testcases/tests/test_models.py (hunk 3 above)
Comment (zkan) [reply api/prontolista/testcases/tests/test_models.py]: Done.

**Pull request merged**